src/main/java/core/basesyntax/FileReaderImpl.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String filePath) throws IOException {
```

Comment:
Remember to follow the encapsulation principle. It's a good practice to hide the implementation details. Consider changing the visibility of the `read` method to be package-private if it's not intended to be used outside the package, or protected if it's supposed to be extended.